static/app/views/issueDetails/groupEventDetails/groupEventDetails.tsx

```diff
@@ -245,4 +251,11 @@ const StyledLayoutSide = styled(Layout.Side)`
   }
 `;
 
+const StyledDataSection = styled('div')`
```

Comment:
This aims to replicate the wrapper we had in `groupEventHeader`:
<img width="578" alt="image" src="https://github.com/getsentry/sentry/assets/44410/71a48f73-d1b4-4276-abf8-f77138f0a2df">